<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringTokenPicker</class>
 <widget class="QDialog" name="StringTokenPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select String</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="labelStringTable">
     <property name="text">
      <string>StringTable:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Filter"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Clear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StringTokenPicker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
